### PR TITLE
CDMConnector support

### PR DESCRIPTION
**Pull request by ginberg**
This PR adds the option to run CohortDiagnostics using CDMConnector. Main changes:
- add function executeDiagnosticsCdm
- add a file DatabaseFunctions that supports DBI db operations
- added a couple functions from CohortGenerator and FeatureExtraction since they used DatabaseConnector functions
- added packages in DESCRIPTION
- added a codeToRunCdm script

### CODE REVIEW

**Comment by ginberg** [comment]
@ablack3 since this is about CDMConnector support, can you have a look at this PR?

File: DESCRIPTION
Comment (cebarboza) [inline DESCRIPTION:1]: Currently the CRAN version is 1.2.1. The latest version I guess still needs some testing and it is in development before going to CRAN.

Comment (ginberg) [reply DESCRIPTION:1]: good point, it's not needed to have the latest dev version, I have set it to 1.2.1 now

**Review by ablack3 — COMMENTED** [review]
Looks good but will require a bit more testing. I'll go ahead an merge though.

**Pull request merged**